backend/src/libs/packages/database/schema/tables-schema.ts

```diff
@@ -1,12 +1,14 @@
 import { relations } from 'drizzle-orm';
 import {
   integer,
+  pgEnum,
   pgTable,
   serial,
   timestamp,
   uniqueIndex,
   varchar,
 } from 'drizzle-orm/pg-core';
+import { ORDER_STATUSES } from 'shared/build/index.js';
```

Comment:
Ok, this is the only place where we'll allow it (direct import from shared)

Reply:
got it